x-pack/plugins/fleet/public/applications/fleet/sections/epm/constants.tsx

```diff
@@ -33,6 +33,7 @@ export const AssetTitleMap: Record<AssetType, string> = {
   map: 'Map',
   data_stream_ilm_policy: 'Data Stream ILM Policy',
   lens: 'Lens',
+  security_rule: 'Detection Engine Rule',
```

Comment:
I stumble over the mismatch of naming and description. Is this something to worry about?

Reply:
updated to "Security Rule", confirming with PM on the naming